frontend_v2/src/app/components/challenge/challengesettings/challengesettings.component.ts

```diff
@@ -406,4 +418,55 @@ export class ChallengesettingsComponent implements OnInit {
     };
     SELF.globalService.showModal(PARAMS);
   }
+
+  // API call to manage the worker from UI.
+  // Response data will be like: {action: "Success" or "Failure", error: <String to include only if action is Failure.>}
```

Comment:
@Kajol-Kumari Can you please use multi line comments in this file also like we are using in other files.